src/main/java/org/folio/des/scheduling/quartz/QuartzExportJobScheduler.java

```diff
@@ -57,6 +57,25 @@ public List<Job> scheduleExportJob(ExportConfig exportConfig) {
     }
   }
 
+  @Override
+  public void deleteJobGroup(String tenantId) throws SchedulerException {
+    try {
+      String ediJobName = getJobGroup(tenantId);
+      Set<JobKey> jobKeySet = scheduler.getJobKeys(GroupMatcher.groupEquals(ediJobName));
+      for (JobKey jobKey : jobKeySet) {
+        scheduler.deleteJob(jobKey);
```

Comment:
scheduler.deleteJobs method can be used to deleted all jobs in call

Reply:
Will be fixed